dspace/config/spring/api/sherpa.xml

```diff
@@ -33,4 +33,14 @@
 		</property>	
 	</bean>
 
+    <bean class="org.dspace.app.sherpa.SHERPAService" id="org.dspace.app.sherpa.SHERPAService">
+        <property name="maxNumberOfTries" value="3"/>
+        <property name="sleepBetweenTimeouts" value="2000"/>
+        <property name="timeout" value="5000"/>
```

Comment:
Could we add some basic comments to describe these settings? For example, I'm assuming `sleepBetweenTimeouts` and `timeout` are both in milliseconds?